core/src/test/java/com/ft/api/ucm/core/datetime/JsonDateTimeSerializerTest.java

```diff
@@ -38,7 +39,7 @@ public void dateTimeIsWrittenAsString() {
       DateTime dateTime = new DateTime();
       jsonDateTimeSerializer.serialize(dateTime, jsonGenerator, serializerProvider);
       verify(mockDateTimeFormatter).format(dateTime);
-      verify(jsonGenerator).writeString(anyString());
+      verify(jsonGenerator).writeString((String) any());

```

Comment:
To be honest I think this tests is semantically wrong, because we have more than one mocks here, but in general this is because [this mock](https://github.com/Financial-Times/unified-content-model/blob/master/core/src/test/java/com/ft/api/ucm/core/datetime/JsonDateTimeSerializerTest.java#L23) is not setup, so from [here](https://github.com/Financial-Times/unified-content-model/blob/master/core/src/main/java/com/ft/api/ucm/core/datetime/BaseJsonDateTimeSerializer.java#L18) this is going to return `null` (because no setup), so is not possible to infer the type from a `null` value at runtime so I'm making sure here at compile time this `null` value is an `String`. Without this the `verify` call fails because the type, not the value. Was this your concern here?

Reply:
Yes, pretty much. Thanks for the explanation. :)